### PR TITLE
[1LP][WIPTEST] Restore edit element nav

### PR DESCRIPTION
**Pull request by digitronik**
Purpose or Intent
=================
In old implementation means before #8462. edit naviagtion:
```
def step(self)
    def step(self):
        self.prerequisite_view.element.edit_element("Text Box")
```
step under steps means fake call. Expected navigation never gone run.

For now, restoring. FA owner need to decide this navigation needed or not. As per my observation its not useful, but I would like FA owner take look on this.

{{pytest: cfme/tests/automate/test_service_dialog.py -k "test_reorder_unsaved_elements or test_reorder_elements" -vv}}

### CODE REVIEW

**Comment by mshriver** [comment]
This either needs to be written as an issue, or resolved in the included commits. Just commenting out the step method entirely is not a 'fix', and without a `step` method there is no purpose to the navigation class existing at all.

File: cfme/automate/dialogs/dialog_element.py [hunk 1]
```diff
@@ -159,7 +160,8 @@ def reorder_elements(self, add_element, second_element, element_data):
             second_element: The second element to be added to the dialog.
             element_data: Already existing first element's data.
         """
-        view = navigate_to(self, 'Edit')
+        navigate_to(self.dialog, 'Edit')
+        view = self.create_view(EditElementView)
```
Comment (mshriver) [inline cfme/automate/dialogs/dialog_element.py]: We shouldn't have to create the view after navigation, it should be returned by navigate_to.  This tells me there is a strange navigation pattern going on in the framework.

This looks like a method that should exist on the `Dialog` entity, not here on the `Element` itself.  Is this editing the dialog, or editing the element?  Why are we expecting an `EditElementView` when we navigated to `Dialog` 'Edit' destination.

Comment (digitronik) [reply cfme/automate/dialogs/dialog_element.py]: @mshriver method better to called widget which is responsible for `element` edit is part of `element` view. but upto now I understand that to edit element
1. need to edit dialog
2. from dialog edit page only we need to edit element. 

so element edit facility should be part of `DialogEditView`. I just avoid this to add normal fix :D. I thing FA owner should take care of major changes related to view. but I am on it. let me try.

Comment (mshriver) [reply cfme/automate/dialogs/dialog_element.py]: Gotcha, yeah I would rather actually fix the view structure problem, either you or the FA owner (ideally working together! :) ) taking on that task of better structuring these views. 

Sounds like EditElementView just needs to be nested in DialogEdit view, or that a conditional view needs to be used.

**Comment by digitronik** [comment]
@mshriver @sshveta @niyazRedhat closing is PR... as to fix those two tests need to restructure... `Collection`, `View` and then `methods` also need to take care about all dependent `tests`. This model page used by most of tests. Its good that FA owner should take care of these failures.

**Comment by sshveta** [comment]
Sounds Good Nikhil . @niyazRedhat Please work on this with @digitronik help.

**Comment by niyazRedhat** [comment]
@sshveta sure

**Pull request closed without merging**